packages/mds-ingest-service/repository/entities/event-annotation-entity.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2021 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BigintTransformer, IdentityColumn, RecordedColumn } from '@mds-core/mds-repository'
+import { Column, Entity, Index } from 'typeorm'
+import { EventAnnotationDomainModel } from '../../@types'
+
+export interface EventAnnotationEntityModel extends IdentityColumn, RecordedColumn {
```

Comment:
We've recently stopped defining entity model properties in terms of the domain model because it creates unnecessary coupling and some libraries that rely on reflection don't always handle the type aliases well. The simplified approach goes something like this:

```ts
@Entity('event_annotations')
export class EventAnnotationEntity extends IdentityColumn(RecordedColumn(class {})) {
  @Column('uuid', { primary: true })
  device_id: UUID

  @Column('bigint', { transformer: BigintTransformer, primary: true })
  timestamp: Timestamp

  @Index()
  @Column('varchar', { length: 255 })
  vehicle_id: string

  ...

}

export type EventAnnotationEntityModel = EventAnnotationEntity
```